app/src/main/java/com/thailam/piggywallet/ui/addtransaction/AddTransactionActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.thailam.piggywallet.ui.addtransaction;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+
+import com.thailam.piggywallet.R;
+
+public class AddTransactionActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_transaction);
+        //
+        initToolbar();
+    }
+
+    private void initToolbar() {
```

Comment:
nếu comment thì em hãy đặt comment vào trước method ntn
cú pháp 
```suggestion
    /** + enter
    private void initToolbar() {
```